sympy/printing/pycode.py

```diff
@@ -526,6 +526,56 @@ def _print_DotProduct(self, expr):
                                self._print(arg1),
                                self._print(arg2))
 
+    def _print_ZeroMatrix(self, expr):
+        return '{}({})'.format(self._module_format('numpy.zeros'),
+            self._print(expr.shape))
+
+    def _print_OneMatrix(self, expr):
+        return '{}({})'.format(self._module_format('numpy.ones'),
+            self._print(expr.shape))
+
+    def _print_FunctionMatrix(self, expr):
+        from sympy.core.function import Lambda
+        from sympy.abc import i, j
+        lamda = expr.lamda
+        if not isinstance(lamda, Lambda):
```

Comment:
Is there ever a case where `FunctionMatrix` is passed a python lambda? In all of the docs and tests,`FunctionMatrix` is only passed a sympy lambda

Reply:
Yup, I asked about this on the sympy gitter chat but didn't get an appropriate response on what I should do. Even though the docs only have examples with SymPy Lambdas, it turns out that it works even with python lamba expressions. Though, if we need only support the ones with SymPy objects then it can just be removed.

Reply:
I don't think FunctionMatrix should be supporting Python lambda. SymPy expressions should have SymPy objects in their args. IMO this block of code should be in the FunctionMatrix constructor.

Reply:
I see. I will remove this from the print function, and if possible someone can make a separate PR adding this to the constructor for `FunctionMatrix` class.